proton

```diff
@@ -1461,8 +1329,8 @@ class Session:
             if not "VKD3D_FEATURE_LEVEL" in self.env:
                 self.env["VKD3D_FEATURE_LEVEL"] = "12_0"
 
-        # enablenvapi beats hidenvgpu
-        if "hidenvgpu" in self.compat_config and "enablenvapi" not in self.compat_config:
+        # hidenvgpu requires disablenvapi to also have been specified
```

Comment:
I think hidenvgpu should probably *imply* disablenvapi now, but I didn't want to change the logic in what's otherwise this straightforward enablenvapi->disablenvapi inversion PR.

Reply:
It may be fine, if we are hiding the gpu vendor nvapi should do no harm. I'll make sure we test that.